README.md

```diff
@@ -101,8 +101,19 @@ Butler is tool to automate front-end development tasks and streamline prototypin
 
   *Note: When you are deploying, Butler will ask you for your GitHub credentials at least once, possibly multiple times. Enter your own GitHub credentials as prompted.*
 
+## Code Linting
 
-### Patching modules
+[GrumPHP](https://github.com/phpro/grumphp) is included for linting and enforcing Drupal code quality standards.  This will run on a Git `pre-commit` hook based on the files you are attempting to commit.  When you run `git commit -m "My message"` GrumPHP will check your code and potentially prevent you from committing until issues are resolved.
```

Comment:
I have... feelings about this... the one other project I'm on that uses pre-commit hooks takes close to a minute to run checks and it's... a pain to work with. I'd much rather run checks in CI, where I don't get yelled at until I'm asking to be, but I'm open to discussion here, as this is definitely a much smaller project and I'd imagine the checks take much less time.

Reply:
I'm certainly up for further discussion on it, but these don't take much time at all in here at the minute (a few seconds).  I would love to have real CI one day...